Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py

```diff
@@ -192,18 +193,41 @@ def get_email_recipients(email_to, email_from, service_mail):
     email_to_set = {email_from}
     email_to = argToList(email_to)
     email_to_set = email_to_set.union(set(email_to))
-    service_mail_recipient = next(recipient for recipient in email_to_set if service_mail in recipient)
-    if service_mail_recipient:
-        email_to_set.remove(service_mail_recipient)
+
+    recipients_to_remove = []
+    for recipient in email_to_set:
+        if service_mail:
+            recipients_to_remove.append(recipient) if service_mail in recipient else None
+        if mailbox:
+            recipients_to_remove.append(recipient) if mailbox in recipient else None
+
+    for recipient_to_remove in recipients_to_remove:
+        email_to_set.remove(recipient_to_remove)
```

Comment:
We should only remove one mailbox, so in case we have the mailbox field configured, remove it, otherwise, remove the service mail if it exists.
```suggestion
    if mailbox and mailbox in email_to_set:
        email_to_set.remove(mailbox)
    elif service_mail and service_mail in email_to_set:
        email_to_set.remove(service_mail)    
   
```